### PR TITLE
Support running multiple osquery instances

### PR DESCRIPTION
**Pull request by RebeccaMahany**
Closes https://github.com/kolide/launcher/issues/1936

Updates the osquery runner to run multiple osquery instances. For now, we only run one -- the "default".

### CODE REVIEW

File: pkg/osquery/runtime/osqueryinstance.go [hunk 1]
```diff
@@ -679,7 +679,7 @@ func calculateOsqueryPaths(rootDirectory string, runId string, opts osqueryOptio
 	// See: https://github.com/kolide/launcher/issues/1599
 	osqueryFilePaths := &osqueryFilePaths{
 		pidfilePath:           filepath.Join(rootDirectory, fmt.Sprintf("osquery-%s.pid", runId)),
-		databasePath:          filepath.Join(rootDirectory, "osquery.db"),
+		databasePath:          filepath.Join(rootDirectory, fmt.Sprintf("osquery-%s.db", runId)),
```
Comment (RebeccaMahany) [inline pkg/osquery/runtime/osqueryinstance.go:682]: TODO -- will move this to a separate PR, there is more work there that needs to be done

**Review by zackattack01 — APPROVED** [review]
